modules/mobile/spec/request/cc_eligibility_request_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/iam_session_helper'
+require_relative '../support/matchers/json_schema_matcher'
+
+RSpec.describe 'CC Eligibility', type: :request do
+  include JsonSchemaMatchers
+
+  let(:rsa_key) { OpenSSL::PKey::RSA.generate(2048) }
+
+  before(:all) do
+    @original_cassette_dir = VCR.configure(&:cassette_library_dir)
+    VCR.configure { |c| c.cassette_library_dir = 'modules/mobile/spec/support/vcr_cassettes' }
+  end
+
+  after(:all) { VCR.configure { |c| c.cassette_library_dir = @original_cassette_dir } }
+
+  before do
+    allow_any_instance_of(IAMUser).to receive(:icn).and_return('9000682')
+    iam_sign_in(build(:iam_user))
+    allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
+  end
+
+  describe 'GET /mobile/v0/community_care/eligibility/:service_type' do
+    context 'valid service type' do
+      context 'eligible service type' do
+        let(:service_type) { 'PrimaryCare' }
+
+        before do
+          VCR.use_cassette('cc_eligibility/get_eligibility_true', match_requests_on: %i[method uri]) do
+            get "/mobile/v0/community_care/eligibility/#{service_type}", headers: iam_headers
+          end
+        end
+
+        it 'returns successful response' do
+          expect(response).to have_http_status(:success)
```

Comment:
Should we test for "true" in the body? Same with false below.

Reply:
haha yes, I forgot to test the most important part.